src/fragments/lib/graphqlapi/native_common/advanced-workflows/common.mdx

```diff
@@ -105,14 +105,14 @@ type Post @model {
 
 type Comment @model {
   id: ID!
-  postID: ID! @index(name: "byPost", sortKeyFields: ["content"])
+  postID: ID! @index(name: "byPost")
```

Comment:
Might be missing something but this edit seems to be related to the API category instead of DataStore. Is this fragment used across both?

Reply:
Updated!